sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -373,7 +374,43 @@ public void waitUntilNumberOfElementsIsReduceToGiven(By selector, int given) {
 
   public String getCheckedOptionFromHorizontalOptionGroup(By options) {
     waitUntilIdentifiedElementIsPresent(options);
-    return baseSteps.getDriver().findElement(options).findElement(SELECTED_RADIO_BUTTON).getText();
+    if (baseSteps.getDriver().findElement(options).findElements(SELECTED_RADIO_BUTTON).size()
+        == 0) {
+      return null;
+    } else {
+      return baseSteps
+          .getDriver()
+          .findElement(options)
+          .findElement(SELECTED_RADIO_BUTTON)
+          .getText();
+    }
+  }
+
+  public boolean isSingleCssInputChecked(By cssInput) {
+    waitUntilIdentifiedElementIsPresent(cssInput);
+    scrollToElement(cssInput);
+    try {
+      boolean value =
+          (baseSteps.getDriver().findElement(cssInput).getAttribute("checked").equals("true"))
+              ? true
+              : false;
+      return value;
+    } catch (Exception e) {
+      return false;
```

Comment:
Please user this method `waitUntilIdentifiedElementHasANonNullValue`, and remove the try catch.
It must be checked or not.

Reply:
could not use that void method to get a boolean return.

my logic here is:
1. get the status checked of the input using isSingleCssInputChecked and parse it in true or null;
2. use the getValueForSingleCssInputIfChecked to parse that result and return the text near the selected element.

I'm doing it by managing the patterns. this will be work as long as the develop is using the same pattern for designing the inputs (they are inherit this pattern in sormas arhitechture) and we are using the ccsSelectors over all the project. this can be extended to other selectors, too.